examples/export_runs.py

```diff
@@ -0,0 +1,50 @@
+from domino import Domino
+import pandas as pd
+import json
+import os
+
+output_dir = "results"
+
+# connect to domino; be sure to have DOMINO_USER_API_KEY env set. Domino automatically includes this for you
+domino = Domino("nick/digits", api_key=os.environ['DOMINO_USER_API_KEY'], host="https://app.dominodatalab.com")
+raw_runs = domino.runs_list()['data']
+
+# print number of runs to STDOUT
+print "Details of {0} runs received".format(len(raw_runs))
+
+# create results directory if it doesn't exist
+if not os.path.exists(output_dir):
+    os.makedirs(output_dir)
```

Comment:
Can you reformat this to using two spaces instead of four for indentation? That's the standard that we tend to use as a team so I'm trying to ensure that even our open source code is consistent in that regard. :)